### PR TITLE
BugFixes

### PR DESCRIPTION
**Pull request by cs-sujeet-ahirrao**
- Modified comments in "Setup Export Template" playbook
- Modified Step name in "Setup Development Env" playbook
- Moved manual input in Start state in "Set GitHub Username" playbook
- Removed comment from "Push Content" playbook
- Updated Condition filter in "Push Changes to Source Control" playbook
- Added Dummy State in "Pull Latest Content from GitHub" playbook
- Added Error Handling in "Merge Pull Request" playbook
- Updated grid Column name "Status to PR Status" in Fetch Open Request (Issue) playbook
- Modified Start State in "Create Pull Request" playbook
- Modified Start State in "Close Change Request" playbook

### CODE REVIEW

File: playbooks/02 - Use Case - CICD/Merge Pull Request.json [hunk 1]
```diff
@@ -170,6 +170,52 @@
             "group": null,
             "uuid": "77bfd50f-41ac-496e-b435-4efbc482a2f7"
         },
+        {
+            "@type": "WorkflowStep",
+            "name": "Pull Request Not Exist",
```
Comment (cs-pooja-singh) [inline playbooks/02 - Use Case - CICD/Merge Pull Request.json]: Does Not Exist

File: playbooks/02 - Use Case - CICD/Merge Pull Request.json [hunk 2]
```diff
@@ -268,13 +314,40 @@
             "group": null,
             "uuid": "d135e606-7153-42f5-88fb-155730aadfba"
         },
+        {
+            "@type": "WorkflowStep",
+            "name": "Is Pull Request Exist",
```
Comment (cs-pooja-singh) [inline playbooks/02 - Use Case - CICD/Merge Pull Request.json]: Does Pull Request Exist

File: playbooks/02 - Use Case - CICD/Pull Latest Content from GitHub.json [hunk 3]
```diff
@@ -807,6 +807,18 @@
             "stepType": "\/api\/3\/workflow_step_types\/0109f35d-090b-4a2b-bd8a-94cbc3508562",
             "group": null,
             "uuid": "fbbe3045-febc-429f-b72e-bfe6856ea13b"
+        },
+        {
+            "@type": "WorkflowStep",
+            "name": "Dummy Step",
```
Comment (cs-pooja-singh) [inline playbooks/02 - Use Case - CICD/Pull Latest Content from GitHub.json:813]: No Operation

File: playbooks/02 - Use Case - CICD/Merge Pull Request.json (hunk 1 above)
Comment (cs-sujeet-ahirrao) [reply playbooks/02 - Use Case - CICD/Merge Pull Request.json]: Done

File: playbooks/02 - Use Case - CICD/Merge Pull Request.json (hunk 2 above)
Comment (cs-sujeet-ahirrao) [reply playbooks/02 - Use Case - CICD/Merge Pull Request.json]: Done

File: playbooks/02 - Use Case - CICD/Pull Latest Content from GitHub.json (hunk 3 above)
Comment (cs-sujeet-ahirrao) [reply playbooks/02 - Use Case - CICD/Pull Latest Content from GitHub.json:813]: Done

**Pull request merged**